auth-web/src/util/constants.ts

```diff
@@ -147,12 +147,14 @@ export enum InvoiceStatus {
 
 export enum AffiliationTypes {
     NAME_REQUEST ='Name Request',
+    AMALGAMATION = 'Amalgamation Application',
     INCORPORATION_APPLICATION = 'Incorporation Application',
     CORPORATION = 'Corporation',
     REGISTRATION = 'Registration'
 }
```

Comment:
It sure would be nice to use https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/filing-names.ts instead, here.

Or perhaps to keep those 5 keys and use FilingsNames.xxx for their values.

Reply:
Sev, the FilingsNames doesn't exist for 'Name Request' and 'Corporation'. Shall I have 3 use the FilingNames while the other their respective string? That looks a bit weird to me having that. Is that OK?

Reply:
Sev, reverting the change. SonarCloud is going crazy about mixing enum values with strings.

Reply:
I could go either way on this.

On the one hand, this is a new enum, different from Filing Names.

On the other hand, there are a lot of common values.

Travis, what do you think?

Reply:
Yeah, in Create or Filings UI, I told SC to ignore that. After all, we know they're all strings, and our Language Service and Linter don't complain.

No worries. Resolving.